docs/features/ulfm.rst

```diff
@@ -107,7 +107,7 @@ please use:
     J. Dongarra: Post-failure recovery of MPI communication
     capability: Design and rationale. IJHPCA 27(3): 244-254 (2013).*
 
-Available from: https://journals.sagepub.com/doi/10.1177/1094342013488238.
+Available from: :doi:`10.1177/1094342013488238`.
```

Comment:
Love it! ❤️

Reply:
I would encourage this `extlinks` feature in `conf.py` for github issues/pr's, generic page links `:ompi-page:..`.
It would allow a consistent and easy changeable documentation path.